### DISCUSSION TITLE
Help for the inexperienced

### DISCUSSION THREAD

**Discussion by Eta-Beta** [moudey/Shell#253] [Q&A]
Good morning, I am not an expert in programming codes but I can do something so I ask, if possible, for help to be able to move the elements of the various menus
as they were in Windows. I also looked at the examples on the site but I just can't figure out how to do it. I have successfully changed the color of the menus. Thanks very much for an answer.

![Screenshot 2023-05-11 100359-ridimensionata](https://github.com/moudey/Shell/assets/104278263/34a89257-e424-47ee-b6cf-ca8cf2ba11da)                                    WINDOWS

![Screenshot 2023-05-11 084339-ridimensionata](https://github.com/moudey/Shell/assets/104278263/f720d2bd-96f6-48cd-a5a2-13cf571aa09d)                                   Nilesoft Shell

**Comment by RubicBG** [comment]
If I guess correctly, you are using the latest release (Shell version 1.8.1). There is a new dev version (Bata - if you want to call it). Тhe syntax is slightly different. If you make changes now, they may not be compatible with the next version. Do you want the release syntax or the dev version?

**Reply by Eta-Beta** [reply]
Hello and thanks a lot for the answer. Unfortunately I'm not able to do that, to write the codes, so I don't understand what you mean.

**Comment by moudey** [comment]
Open the shell.nss file and edit this line in the static section by adding a `//` sign at the beginning of the line to disable it.
```
item(where=this.title.length > 25 menu=title.more_options)
```
to
```
// item(where=this.title.length > 25 menu=title.more_options)
```
Save changes, press <kbd>CTRL</kbd> + <kbd>RIGHT-CLICK</kbd> to reload settings.

**Reply by Eta-Beta** [reply]
Hello and thanks for the answer. Perfect I had to restart the PC though, because the settings weren't updated.

![Screenshot 2023-05-12 214545-2](https://github.com/moudey/Shell/assets/104278263/96abb09f-b202-4f66-8778-2682187f4aed)

Thanks again. The menu is still long, but I should be able to disable Defender UI, Scan with Microsoft Defender and Create Quick Backup... from the software settings. I had read in another post of yours that you could disable an item from Nilesoft Shell but I didn't understand the changes.

**Reply by Eta-Beta** [reply]
I have found my answer  #173 . Ok for the desktop but what are the other menus called? sorry for the maybe stupid question.
